.github/workflows/build-and-test.yml

```diff
@@ -32,12 +32,14 @@ jobs:
     
     - name: Build the mod
       run: ./gradlew build
-        
-    - uses: "marvinpinto/action-automatic-releases@latest"
-      with:
-        repo_token: "${{ secrets.GITHUB_TOKEN }}"
-        automatic_release_tag: "latest"
-        prerelease: true
-        title: "Latest Development Build"
-        files: build/libs/*.jar
-            
+
+    - name: Run server for 1 minute
```

Comment:
is 1m enough to start runServer, or is that "1m after finished loading"

Reply:
It should be plenty, if you don't use tons of dependencies